### ISSUE TITLE
.net framework project can't load rdotnet targeting netstandard2.0

### ISSUE THREAD

**Issue by TongZhai** [rdotnet/rdotnet#119]
I recompiled RDotNet.dll from source on netstandard2.0, then I copy the dll into my .net framework project (4.5.2 and 4.7.2) output folder to be called upon to run R script. But it failed to load during program launch, the error message is as follow:

System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.

I don't get this problem when my project was on .net framework 3.5.

I needed to recompile my own RDotNet.dll due to the need for the searching current user registry update I posted. Any suggestions?

**Comment by lrasmus** [comment]
I think it makes sense that the .NET 4.5.2 code wouldn't work, since [it looks like .NET Standard 2.0 requires .NET 4.6.1 or higher](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support).  But, that means your 4.7.2 version should work.

Is there any more information on what is trying to be loaded via reflection?  I took a peek through the R.NET code and can see a few spots where `System.Reflection` is being used.  [This SO post](https://stackoverflow.com/a/46562003/5670646) makes me think it may not be possible to mix the two different approaches to reflection?

In a pinch, this [very unofficial](https://github.com/stattag/rdotnet) fork still uses the .NET Framework, but isn't kept entirely in sync with the main repo.

**Comment by TongZhai** [comment]
I was able to make the netstandard2.0 DLL work with my .net framework project. What I did was the following: 1) make my project to be .net 4.7.2; 2) trace the dependency of the rdotnet netstandard2.0 DLL, it needs `Dynamicinterop` and `Microsoft.Win32.Registry` DLLs, once these two were added into my .net framework project's dependencies, it works now. So, I did NOT have to re-compile `RDotNet.dll` to .net framework.
